### ISSUE TITLE
Does Travis CI need to build a docker image?

### ISSUE THREAD

**Issue by okaycj** [lookit/lookit-api#729]
# TL;DR
Pulling images from docker hub might be taking too much time and wasting Travis CI credits.

# Narrative
Currently, Travis CI builds a docker image and installs our app in it.  The process of pulling down the python image is time consuming and wasting our gifted credits on travis-ci.com.  Additionally, docker hub is limiting downloads for logged in users, causing false fail reporting.  This would effect the Travis CI badges we have in our readme.  

Could we not build the image?

Alternatively, we could just install the necessary python packages in the travis ci provided python environment and run the test suite.  We could also add call to `pre-commit` to verify that those checks are passing as well.
